frontend/packages/process-editor/src/components/ConfigPanel/EditDataType/EditDataType.test.tsx

```diff
@@ -74,25 +91,41 @@ describe('EditDataType', () => {
   it('should display the existing data type in preview as a button to edit when task has connected data model', async () => {
     const user = userEvent.setup();
     const existingDataType = mockBpmnApiContextValue.layoutSets.sets[0].dataType;
-    renderEditDataType();
+
+    renderEditDataType({
+      componentProps: {
+        existingDataTypeForTask: existingDataType,
+        datamodelIds: [existingDataType],
+      },
+    });
+
     const updateDataTypeButton = screen.getByRole('button', {
       name: textMock('process_editor.configuration_panel_set_datamodel'),
     });
     expect(screen.getByText(existingDataType)).toBeInTheDocument();
 
     await user.click(updateDataTypeButton);
+
     const nativeSelect = screen.getByRole('combobox', {
       name: textMock('process_editor.configuration_panel_set_datamodel'),
     });
+
     expect(nativeSelect).toHaveValue(existingDataType);
   });
 
   it('should display the existing data type in preview when clicking the close button after edit mode and task has data type', async () => {
     const user = userEvent.setup();
-    renderEditDataType();
+    renderEditDataType({
+      componentProps: {
+        hideSelectDeleteButton: false,
```

Comment:
Hm, not sure that this is needed to be set to false since it is the `close` button that is clicked here and not the delete 🤔

Reply:
I guess since the component defaults it to false, it is not even needed as a prop in the defaultProps?